pkg/internal/testing/controlplane/apiserver.go

```diff
@@ -123,9 +133,12 @@ func (s *APIServer) setProcessState() error {
 		return err
 	}
 
-	s.processState.Args, err = internal.RenderTemplates(
-		internal.DoAPIServerArgDefaulting(s.Args), s,
-	)
+	args := s.Args
+	if len(args) == 0 {
+		args = APIServerDefaultArgs
+	}
```

Comment:
Should we opt to merge the Args given from users with the default one (and potentially overwriting some)?

Reply:
this PR is supposed to generally not change behavior from what happened previously, and this non-merging behavior is what the existing code does.